dmoj/tests/test_checker.py

```diff
@@ -63,3 +65,20 @@ def test_sorted(self):
         assert check(b'1 2\n3', b'3\n1 2')
         assert not check(b'1 2\n3', b'3\n2 1')
         assert check(b'1 2\n3', b'3\n2 1', split_on='whitespace')
+
+    def test_linematches(self):
+        from dmoj.checkers.linematches import check
+
+        def validate(res, expected_points, passed):
```

Comment:
This is not the right style of testing. Checks should be done with `self.assert_*`. This can be fixed later given how broken this file is.